src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-sentry_sdk.py

```diff
@@ -17,4 +17,14 @@
                  "sentry_sdk.integrations.modules",
                  "sentry_sdk.integrations.argv",
                  "sentry_sdk.integrations.logging",
-                 "sentry_sdk.integrations.threading"]
+                 "sentry_sdk.integrations.threading",
+                 "sentry_sdk.integrations.django",
+                 "sentry_sdk.integrations.flask",
+                 "sentry_sdk.integrations.bottle",
+                 "sentry_sdk.integrations.falcon",
+                 "sentry_sdk.integrations.sanic",
+                 "sentry_sdk.integrations.celery",
+                 "sentry_sdk.integrations.rq",
+                 "sentry_sdk.integrations.aiohttp",
+                 "sentry_sdk.integrations.tornado",
+                 "sentry_sdk.integrations.sqlalchemy"]
```

Comment:
Normally we'd use:
```python
from PyInstaller.utils.hooks import collect_submodules
hiddenimports = collect_submodules("sentry_sdk.integrations")
```
instead of big lists of submodules although I notice the original didn't do that either. Did you try `collect_submodules()`? Does it not work?

Reply:
Just tried. Doesn't work.
First there is a python error while packaging (`ModuleNotFoundError: No module named 'django'`),but continues and when running executable `ModuleNotFoundError: No module named 'sentry_sdk.integrations.stdlib'
[22269] Failed to execute script __main__`